<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionWindow</class>
 <widget class="QDialog" name="ConversionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Conversion</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Float/Double ↔ Hex</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>211</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Float:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_floatFloat">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_hexToFloat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>▲</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hex:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QRadioButton" name="radioButton_doubleHex">
       <property name="text">
        <string>Double</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Precision:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_toFloatHex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>▼</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QRadioButton" name="radioButton_floatHex">
       <property name="text">
        <string>Single</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_floatHex">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Little Endian ↔ Big Endian</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_int16">
       <property name="text">
        <string>Int16</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_int32">
       <property name="text">
        <string>Int32</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButton_int64">
       <property name="text">
        <string>Int64</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_LE_BE_in"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_LE_BE_out"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Reverse byte order</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
